caterer/src/main/java/com/vendor/caterer/model/FoodItem.java

```diff
@@ -1,10 +1,22 @@
 package com.vendor.caterer.model;
 
-import lombok.Data;
+import org.springframework.data.elasticsearch.annotations.Document;
 
 import java.math.BigDecimal;
-@Data
-public class FoodItem extends MenuItem {
-    private Boolean isVeg;
-    private BigDecimal pricePerQuantity;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+
+@Document(indexName = "food-item")
+public class FoodItem {
+    private UUID id;
+    private String name;
+    private String description;
+    private BigDecimal price;
```

Comment:
For case of drinks there might be different price for different volume so need handle that scenario for drinks

Reply:
during the discussion, we have assumed that the price for every item (including drinks) will per person.